docs/integration/spring-cloud-gateway.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Spring Cloud Gateway
+description: Using Casdoor in Spring Cloud Gateway
+keywords: [Sping Cloud Gateway]
+---
+
+[casdoor-springcloud-gateway-example](https://github.com/casdoor/casdoor-springcloud-gateway-example) is an example on how to use [casdoor-spring-boot-starter](https://github.com/casdoor/casdoor-spring-boot-starter) as a OAuth2 plugin in Spring Cloud Gateway. We will show you the steps below.
+
+## Step1. Deploy Casdoor
+
+Firstly, the Casdoor should be deployed.
+
+You can refer to the Casdoor official documentation for the [Server Installation](/docs/basic/server-installation). Please deploy your Casdoor instance in **production mode**.
+
+After a successful deployment, you need to ensure:
+
+- Open your favorite browser and visit **http://localhost:8000**, you will see the login page of Casdoor.
+- Input `admin` and `123` to test login functionality is working fine.
+
+Then you can quickly implement a casdoor based login page in your own app with the following steps.
+
+## Step2: Init a Spring Cloud Gateway
+
+You can use the code of this example directly or combine your own business code.
+
+We need a gateway service and at least one business service.
+
+In this example, `casdoor-gateway` as the gateway service and `casdoor-api` as the business service.
+
+## Step3: Include the dependency
+
+Add `casdoor-spring-boot-starter` to the Spring Cloud Gateway project.
+
+For Apache Maven:
+
+```Maven
+<!-- https://mvnrepository.com/artifact/org.casbin/casdoor-spring-boot-starter -->
+<dependency>
+    <groupId>org.casbin</groupId>
+    <artifactId>casdoor-spring-boot-starter</artifactId>
+    <version>1.x.y</version>
+</dependency>
+```
+
+For Gradle:
+
+```gradle
```

Comment:
Please use `groovy` instead of `gradle`

Before that, please modify `docusaurus.config.js`
add `groovy` in `additionalLanguages`
like:
```js
additionalLanguages: ['nginx', 'groovy'],
```